src/components/SoftwareWalletWarningMessage.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+
+
+/**
+ * Component that shows a warning message if user decides to use a software wallet
+ *
+ * @memberof Components
+ */
+const SoftwareWalletWarningMessage = (props) => {
+  return (
+    <div>
+      <p className="mt-4">Using a software wallet is not the safest method to store your tokens. Your data is exposed to an attack because it's connected to the internet.</p>
+      <p className="mt-4">In case you want a safe method, you should have a hardware wallet (e.g. Ledger) and use it.</p>
```

Comment:
Suggestion: "If you want a safer method, you can use a hardware wallet. We currently have support for Ledger. Check out more information [here](link)". And I think we should have a page giving more details in our website.

This alert is important but (i) we don't want to panic our users, and (ii) the hardware wallets are not bullet proof. Token custody is a hard problem for all cryptos and we should make it clear that we are just giving a suggestion.

Reply:
Done